ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,313 @@
+
+        highlightedBackgroundColor = appearanceProxy.backgroundColor.pressed
+
+        borderSize = appearanceProxy.borderSize
+        disclosureInterspace = appearanceProxy.disclosureInterspace
+        disclosureSize = appearanceProxy.disclosureSize
+        horizontalCellPadding = appearanceProxy.horizontalCellPadding
+        iconInterspace = appearanceProxy.iconInterspace
+        iconSize = iconStyle == MSFListIconVnextStyle.large ? appearanceProxy.iconSize.large : appearanceProxy.iconSize.default
+        subtitleFont = appearanceProxy.sublabelFont
+        textFont = appearanceProxy.labelFont
+    }
+}
+
+public struct MSFListView: View {
+    @ObservedObject var state: MSFListVnextState
+    @ObservedObject var tokens: MSFListTokens
+    var sections: [MSFListVnextSectionData]
+
+    public init(sections: [MSFListVnextSectionData],
+                iconStyle: MSFListIconVnextStyle) {
+        self.sections = sections
+        self.state = MSFListVnextState()
+        self.tokens = MSFListTokens(iconStyle: iconStyle)
+    }
+
+    public var body: some View {
+        List {
+            ForEach(sections, id: \.self) { section in
+                if section.title != nil {
+                    if #available(iOS 14.0, *) {
+                        Section(header: Header(title: section.title ?? "", tokens: tokens)) {}
+                            .textCase(.none)
+                            .listRowInsets(EdgeInsets())
+                            .padding(.top, tokens.horizontalCellPadding / 2)
+                            .padding(.leading, tokens.horizontalCellPadding)
+                            .padding(.trailing, tokens.horizontalCellPadding)
+                            .background(Color(tokens.backgroundColor))
+                    } else {
+                        Text(section.title ?? "")
+                            .listRowInsets(EdgeInsets(
+                                    top: tokens.horizontalCellPadding / 2,
+                                    leading: tokens.horizontalCellPadding,
+                                    bottom: tokens.horizontalCellPadding / 2,
+                                    trailing: tokens.horizontalCellPadding))
+                            .font(Font(tokens.subtitleFont))
+                            .foregroundColor(Color(tokens.subtitleColor))
+                    }
+                }
+                ForEach(section.cells, id: \.self) { cell in
+                    MSFListCellView(cell: cell, layoutType: section.layoutType, tokens: tokens)
+                        .border(state.hasBorder ? Color(tokens.borderColor) : Color.clear, width: state.hasBorder ? tokens.borderSize : 0)
+                        .frame(maxWidth: .infinity)
+                }
+            }
+        }
+        .environment(\.defaultMinListRowHeight, 0)
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+}
+
+extension MSFListView {
```

Comment:
What's the benefit of having these types in the MSFListView extension?

Reply:
It was for organization purposes, but I'm thinking of splitting the file (which will be tracked in a GH issue)

Reply:
https://github.com/microsoft/fluentui-apple/issues/364